l1-contracts/contracts/governance/ChainAdmin.sol

```diff
@@ -21,6 +33,25 @@ contract ChainAdmin is IChainAdmin, Ownable2Step {
     /// @dev Needed for the offchain node administration to know when to start building batches with the new protocol version.
     mapping(uint256 protocolVersion => uint256 upgradeTimestamp) public protocolVersionToUpgradeTimestamp;
 
+    EnumerableSet.AddressSet internal activeRestrictions;    
+
+    function getRestrictions() public view returns (address[] memory) {
+        return activeRestrictions.values();
+    }
+
+    function isRestrictionActive(address restriction) external view returns (bool) {
```

Comment:
I know it is WIP, sorry for nits :)
Use the `_` prefix convention

Reply:
it is an external method, no need to use `_` there, but I'll look through internal ones